core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1201,16 +1201,25 @@ abstract class RDD[T: ClassTag](
     jobResult
   }
 
+  def treeAggregate[U: ClassTag](zeroValue: U)(
+    seqOp: (U, T) => U,
+    combOp: (U, U) => U,
+    depth: Int = 2): U = withScope {
+    treeAggregate(zeroValue, seqOp, combOp, depth, finalAggregateOnExecutor = false)
+  }
+
   /**
    * Aggregates the elements of this RDD in a multi-level tree pattern.
    * This method is semantically identical to [[org.apache.spark.rdd.RDD#aggregate]].
    *
```

Comment:
Isn't this method doc for the above API? Also can we add a new scala doc for the newly added method accordingly?